addons/sale/models/sale_order.py

```diff
@@ -3,17 +3,14 @@
 
 from datetime import timedelta
 from itertools import groupby
-from markupsafe import Markup
 
 from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.addons.payment import utils as payment_utils
 from odoo.exceptions import AccessError, UserError, ValidationError
 from odoo.fields import Command
-from odoo.osv import expression
 from odoo.tools import float_is_zero, format_amount, format_date, html_keep_url, is_html_empty
 from odoo.tools.sql import create_index
 
-from odoo.addons.payment import utils as payment_utils
-
```

Comment:
Cosmetic changes to remove for a PR in stable version.

Furthermore, moving `payment_utils` import goes against our [guidelines](https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports)

Reply:
Indeed, my IDE "optimized" the imports, sorry about that :sweat_smile: